variants/esp320/pins_arduino.h

```diff
@@ -9,6 +9,12 @@
 #define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
 #define digitalPinHasPWM(p)         (p < 34)
 
+static const uint8_t LED_BUILTIN = 5;
+static const uint8_t BUILTIN_LED = 5;
```

Comment:
since those are the same, wouldn't it be better if one of them is define to forward to the other one?